.travis.yml

```diff
@@ -1,7 +1,6 @@
 language: groovy
 matrix:
   include:
-  - jdk: openjdk7
```

Comment:
I'm all for dropping JDK7 support (it is well into later EOL phases), and this is in-line with the comment to bump the version-number to 3.0.  Maybe we can create a 2.1.x branch to provide a slightly longer sunset for old-version support and release a 2.1.1.

This change has follow-on container support/deprecation:
- Jetty: Only Jetty 9.3.x and 9.4.x are JDK8+.  Of course Jetty can and does work on later JDKs.  Jetty 9.2.x and earlier should probably go.  Jetty 7 and 8 are well-and-truely EOL anyway and should become unsupported.
- Tomcat: A little trickier because Tomcat specifically supports "JDK? and later" and Tomcat 9 is the first to require minimum JDK8+.  I understand that TC6 is the current newest EOL release, and we don't support/test that.

I proposed that we raise either a meta-issue or a set of issues (mostly for communication and tracking) to:

- Move master to 3.0
- Deprecate all JDK7 support
- Deprecate Jetty 7 and Jetty 8 support (and check 9.2.x)
- (Optional?) Maybe retire old Gradle support if it helps remove some code and complexity.

Users requiring these will need to run a 2.x release.

References:
- https://www.java.com/en/download/faq/java_7.xml
- https://www.eclipse.org/jetty/documentation/current/what-jetty-version.html
- http://tomcat.apache.org/whichversion.html